toastify/toastify.nuspec

```diff
@@ -4,9 +4,9 @@
   <metadata>
    <id>toastify</id>
    <title>toastify</title>
-   <version>1.7.8</version>
+   <version>1.8.3</version>
    <authors>Geoffrey Huntley</authors>
-   <owners>ghuntley</owners>
+   <owners>ghuntley, slayerice09</owners>
```

Comment:
A while ago I send a message through the site to the "maintainer" (guessing you?) saying there was an update, but didn't hear or see anything, so yesterday (following the site's rules) I used the "contact the site admin" saying there was a new update. They replied saying either I become a maintainer or they remove it... If you're still maintaining it, I'm happy to back off...